charts/shinyproxy/templates/configmap.yaml

```diff
@@ -6,6 +6,15 @@ metadata:
     {{- include "library-chart.labels" . | nindent 4 }}
 data:
   application.yml: |-
+    {{ if .Values.monitor.enabled }}
+    usage-stats-micrometer-prefix: {{ .Values.monitor.micrometer.prefix }}
+    usage-stats-url: micrometer
```

Comment:
is micrometer  the only choice possible ? shouldn t  it be a param ? ( idk)

Reply:
not the only choice, but the docker image itself is not very well design so we do the strict minimum here